src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1732,12 +1732,6 @@ This version used in: {2}</value>
   <data name="Implement_implicitly" xml:space="preserve">
     <value>Implement implicitly</value>
   </data>
-  <data name="Implement all interfaces explicitly" xml:space="preserve">
-    <value>Implement all interfaces explicitly</value>
-  </data>
-  <data name="Implement explicitly" xml:space="preserve">
-    <value>Implement explicitly</value>
-  </data>
```

Comment:
@CyrusNajmabadi Accident?

Reply:
No.  Fixed up to make the name `Implement_explicitly` instead of `Implement explicitly`.